ImperatorToCK3/CK3/World.cs

```diff
@@ -47,10 +50,18 @@ public class World {
 			Logger.Info("Loading map data...");
 			MapData = new MapData(config.CK3Path);
 
-			// Scraping localizations from Imperator so we may know proper names for our countries.
+			// Scrape localizations from Imperator so we may know proper names for our countries.
 			locDB.ScrapeLocalizations(impWorld.ModFS);
-
-			// Loading Imperator CoAs to use them for generated CK3 titles
+			
+			// Load CK3 religions from game and blankMod
+			var blankModReligionPath = Path.Join("blankMod", "output", "common", "religion");
+			Religions.LoadHolySites(ck3ModFS);
+			Religions.LoadHolySites(Path.Combine(blankModReligionPath, "holy_sites"));
```

Comment:
It's possible to be sneaky and create a fake mod that just points to the blankMod stuff. Though that may not be helpful depending on the load order you intend.

Reply:
Thanks for the idea, seems to be working. Currently I don't add any other mod to ck3ModFS anyway.